plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPropertyPage.java

```diff
@@ -69,18 +72,20 @@ protected Control createContents(Composite parent) {
       return container;
```

Comment:
Should the BlankDeployPreferencesPanel be returned here since it has the appropriate error message?